.github/workflows/v3_mcp-student-prd.yml

```
@@ -0,0 +1,93 @@
+  IMAGE_NAME: ${{ github.repository }}/mcp-student
+
+jobs:
+  build-arm:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-24.04-arm
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)